integrations/clickup/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,51 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: team
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupTeam"'
+          identifier: .id
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .id + \"/home\""
+            member: (.members | map(.user.username | tostring))
+  - kind: project
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupProject"'
+          identifier: .id
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .__team_id + \"/v/b/li/\" + .id "
+            startDate: if .start_date != null then (.start_date | todate) else "946684800" | todate  end  # Unix epoch for 01/01/2000
+            endDate: if .due_date != null then (.due_date | todate) else "946684800" | todate end  # Unix epoch for 01/01/2000
```

Comment:
Why are you hardcoding 946684800? If there is no start date then there shouldn't be any value for it

Reply:
I have fixed this, it is now empty when not present